src/containers/CourseResultViewContainer.js

```diff
@@ -97,7 +123,100 @@ function CourseResultViewContainer() {
   useEffect(() => {
       window.scrollTo(0, 0);
   } ,[]);
+
+  useEffect(() => {
+      async function fetchData() {
+        setIsFetchingCourseStatus(true);
+        const data = await dispatch(getCourseEnrollInfo(isCourseStatusModalOpen));
```

Comment:
Should I add try-catch block here in case there are some errors appear when calling API?

Reply:
Sure, will add it after I finish my meal. 👍

Reply:
Done~